extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/FSInternetRadioDiscoveryParticipant.java

```diff
@@ -100,28 +100,13 @@ public DiscoveryResult createResult(RemoteDevice device) {
                 }
 
                 final DiscoveryResult result = DiscoveryResultBuilder.create(uid).withProperties(properties)
-                        .withLabel(getLabel(device)).build();
+                        .withLabel(device.getDisplayString()).build();
```

Comment:
Why do you remove the fallback to `details.getModelDetails().getModelDescription();` and `details.getFriendlyName();`?